automated install/basic-install.sh

```diff
@@ -1023,106 +1179,119 @@ setDNS() {
 
 # Allow the user to enable/disable logging
 setLogging() {
-    # Local, named variables
-    local LogToggleCommand
-    local LogChooseOptions
-    local LogChoices
-
-    # Ask if the user wants to log queries
-    LogToggleCommand=(whiptail --separate-output --radiolist "Do you want to log queries?" "${r}" "${c}" 6)
-    # The default selection is on
-    LogChooseOptions=("On (Recommended)" "" on
-        Off "" off)
-    # Get the user's choice
-    LogChoices=$("${LogToggleCommand[@]}" "${LogChooseOptions[@]}" 2>&1 >/dev/tty) || (printf "  %bCancel was selected, exiting installer%b\\n" "${COL_LIGHT_RED}" "${COL_NC}" && exit 1)
-    case ${LogChoices} in
-        # If it's on,
-        "On (Recommended)")
-            printf "  %b Logging On.\\n" "${INFO}"
-            # set the GLOBAL variable setting to true
+    # Ask the user if they want to enable logging
+    dialog --no-shadow --keep-tite \
+        --backtitle "Pihole Installation" \
+        --title "Enable Logging" \
+        --yesno "\\n\\nWould you like to enable query logging?" \
+        "${r}" "${c}" && result=0 || result=$?
+
+    case ${result} in
+        "${DIALOG_OK}")
+            # If they chose yes,
+            printf " %b Query Logging on.\\n" "${INFO}"
             QUERY_LOGGING=true
             ;;
-        # Otherwise, it's off,
-        Off)
-            printf "  %b Logging Off.\\n" "${INFO}"
-            # set the GLOBAL variable setting to false
+        "${DIALOG_CANCEL}")
+            # If they chose no,
+            printf " %b Query Logging off.\\n" "${INFO}"
             QUERY_LOGGING=false
             ;;
+        "${DIALOG_ESC}")
+            # User pressed <ESC>
+            printf " %bEscape pressed, exiting installer at Query Logging choice.%b\\n" "${COL_LIGHT_RED}" "${COL_NC}"
```

Comment:
```suggestion
            printf "  %b Escape pressed, exiting installer at Query Logging choice.%b\\n" "${COL_LIGHT_RED}" "${COL_NC}"
```

Reply:
Should this have two additional spaces?

Reply:
It should have 2 spaces before `%b`.

Most entries have one space after too, but not all.
```sh
printf "  %b Checking for / installing Required dependencies for Pi-hole software...\\n" "${INFO}"
```
```sh
printf "  %bError: URL %s/%s not found%b\\n" "${COL_LIGHT_RED}" "${url}" "${binary}" "${COL_NC}"
```


Maybe we can define a standard.

Reply:
Oh, okay. All the ones like that are the `Escape` lines. I'm fine with them matching the style that was already there.